### ISSUE TITLE
Change the logo in the new Project Manager to one of the official versions

### ISSUE THREAD

**Issue by peterhoglund** [godotengine/godot-proposals#9065]
### Describe the project you are working on

-

### Describe the problem or limitation you are having in your project

The new Project Manager in 4.3 is a great update, however the logo in the upper left corner is not good.
1. An outlined logo is never used under any other circumstances, meaning the first logo shown in the product is not official.
3. The small size of the logo makes it look messy to the point where it distorts the visual features of the official logo, especially the eyes where the pupils are missing.
4. There is no reason to use a new version of the logo when there is already an official black/white version that can be used here.

Current version
![image](https://github.com/godotengine/godot-proposals/assets/522470/b7309ff2-6f38-4f80-b47a-09fde6db2c45)

Suggested update
![image](https://github.com/godotengine/godot-proposals/assets/522470/33507315-2191-4d3a-b831-236b02ffdc25)

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

See image above

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

-

### If this enhancement will not be used often, can it be worked around with a few lines of script?

-

### Is there a reason why this should be core and not an add-on in the asset library?

-

**Comment by peterhoglund** [comment]
I agree with the critique from Menica, the monochrome logo does have a disturbing look to it :) Personally, I'm not a fan of outlined logos because they tend to break apart at small sizes, which this is a good example of. So, I don't think adding an outlined version to the official logo variations is a good idea, which we are doing now. Since this logo is now in the product this is an official version of the logo. And worse, from a brand point of view, is that we create custom logo variations like this on a whim.

If the monochrome version looks weird, the colored version is perfectly fine in this scenario, imo.
<img width="525" alt="image" src="https://github.com/godotengine/godot-proposals/assets/522470/6f320f1d-8d18-4079-b820-240e400587f3">

**Comment by YuriSizov** [comment]
> If the monochrome version looks weird, the colored version is perfectly fine in this scenario

Please try it at its actual final size for your mockup. Most of the lines in this screenshot are going to be subpixel for most common resolutions. The irises, for example, were previously present in the logo used for the project manager, and they were barely a full pixel in size. And lines in your mockup are sometimes smaller than a half of that.

The current version was specifically adjusted to be clearly readable at a small scale, with extra elements removed and thickness added where needed. The full standard version of the logo will require similar chances, which would also disturb its "brand look". The logo shape is too complex which makes it not infinitely scalable. It cannot be used as is at the resolution used in the updated project manager.

The project doesn't have a brandbook or strict guidelines yet. It is not outlandish to experiment with what the project has and see what is needed to make things work. This can help define the brand guidelines in the future.

**Comment by peterhoglund** [comment]
> The current version was specifically adjusted to be clearly readable at a small scale
I'm sorry, but from a graphical design perspective, it doesn't hold up. It is just not very well made. What does it look like in large scale? The outlines will be way too thick. And since it's just a center outline the thickness of the lines covers the face, which makes it lose its proportions. 

> The logo shape is too complex which makes it not infinitely scalable. It cannot be used as is at the resolution used in the updated project manager.
Either, don't use the logo in that spot or make the logo larger so it does not break. Those are two reasonable options. The least reasonable option is to create a new version that doesn't share the defining features of the original only to make it fit your own constraints.

> It is not outlandish to experiment with what the project has and see what is needed to make things work
Yes, it is outlandish to prototype logo ideas on a hero-spot in the live product. Even if there is no official brand style guide the robot logo is very recognizable by now given all the attention Godot has gotten lately, and should be treated with care.

**Comment by YuriSizov** [comment]
> which makes it lose its proportions.

That is kind of the point of adapting something to work on the smaller scale. You exaggerate the proportions of the most recognizable elements. You make sure that the silhouette is still recognizable. You don't try to preserve every detail, because that just doesn't work in practice. This is not print, the resolution is finite and at a certain point details stop to matter and stop to be defined.

> It is just not very well made.

Yes, it is not well made. But you need to consider what you suggest in a way that would make it work better. Suggesting to use one of the icons we already have just because we already have them is not practical. These variations while iconic suffer from a number of issues with scale, contrast, and adaptiveness. This is highlighted by the thread linked above by Calinou, which has been referenced for this project manager rework.

> The least reasonable option is to create a new version that doesn't share the defining features of the original

It shares the same silhouette, which is the most important and the most recognizable feature of the project's logo. The exact shape is slightly different, but it is expected at this scale because physical limitations restrict allowed precision. One also must be aware of pixel alignment, which is tricky for such a complex shape. Care has been taken to preserve the logo while making it work for the new placement.

> only to make it fit your own constraints.

Well, the project didn't have the need for this kind of logo, and now it does. This is the first draft, and it can be improved upon. The project doesn't have a design team to task them with adapting the logo for this placement. Everything, including the logo itself and its numerous adjustments over the years, is created by shared efforts from those who are willing.

This is not the same sensibility you might expect from a proprietary project developed behind closed doors or with a dedicated design team. But that's open source for you, you make do with what you have. We try and then we make changes based on the feedback.

> in the live product

This is not a live product. It's a dev snapshot of a future version of the engine. It's not set in stone, and this design direction is experimental. This is how things get done in Godot, with public testing and discussion. Anything and everything can be changed before the release.

A new precedent for the logo has been created, and a new variation has been made to demonstrate and support the case. Now it can be iterated upon and defined. Do we want a logo at all? Do we want to have a picture or just the text, or the other way around? Can other adjustments be made to the style and the implementation of it? The community will tell the contributors. The answers to these questions will inform the team what kind of graphics we need, and then it can be made properly.

**Comment by peterhoglund** [comment]
>You exaggerate the proportions of the most recognizable elements

You are right in theory, however you would never do that to a logo. It is considered very bad practice to distort an established logo like that. What you are describing is something you do to icons. For example, in a menu or a button. 

> But you need to consider what you suggest in a way that would make it work better

You are right that my initial suggestion of simply replacing the new variation with any of the existing logos might not be a viable solution, I was quick to make a point about not creating a new version of the logo. But I did suggest in my previous post to instead either make the logo larger or not have it in that spot at all.

>Well, the project didn't have the need for this kind of logo, and now it does

It does not. You created the need for this in your design. It is an introduced problem that can be avoided by changing the design.

>But that's open source for you, you make do with what you have. We try and then we make changes based on the feedback

I'm not sure what you mean, this is my feedback and this Issue is my opinion on the matter. Isn't this exactly according to how things are supposed to be handled?

>A new precedent for the logo has been created, and a new variation has been made to demonstrate and support the case. Now it can be iterated upon and defined. Do we want a logo at all? Do we want to have a picture or just the text, or the other way around? Can other adjustments be made to the style and the implementation of it? The community will tell the contributors. The answers to these questions will inform the team what kind of graphics we need, and then it can be made properly.

Again, I don't understand your point. Isn't this exactly what I am doing now? I'm trying to express, through the means available to the community and before 4.3 is officially launched, that I think it is a bad idea to introduce this logo variation in the live product. I think the need for this variation is forced and not necessary and that it strays too far away from the established logo. It would be better to work _with_ the not-so-great qualities of the original logo by, for example, removing the logo or making the space for the logo larger. Since we have full control over the design of the Project Manager, this is doable. 

I think you and I are attacking this problem from different angles. I get the feeling you see this from a technical perspective – the area that happened to be available for the logo is so small that the original logo won't look good. So we need to fix it with a logo that can be placed within the pixels we have available. I see this from a designer's perspective – We have an established logo, let's make enough space for it so it can be presented in a way that looks good and is recognizable.

I'll try to be even more concrete, here are a few suggestions, without making any big changes to the overall layout, to illustrate what I mean:

The most obvious solution is to simply make the top area a bit larger to fit the logo in a size that will not break.
![image](https://github.com/godotengine/godot-proposals/assets/522470/291046d0-2473-4b76-b4b4-d3fed78f942d)

Another solution is to have it to the right
![image](https://github.com/godotengine/godot-proposals/assets/522470/085920f1-f917-4955-a868-1df9b92950cf)
![image](https://github.com/godotengine/godot-proposals/assets/522470/1329a562-0dfc-4341-b1f8-e4ac2d70afb3)

Or simply remove the logo from the Project Manager. The logo has just been shown in the splash screen and there will be plenty of logos when the user starts creating projects in the form of the project icon. We can show it in the empty state since this will be the first thing a new user sees
![image](https://github.com/godotengine/godot-proposals/assets/522470/71dae375-d871-4a3f-b965-54231c4874fc)

**Comment by YuriSizov** [comment]
> Again, I don't understand your point.

Sorry, this was not a critique of your feedback. What I wanted to convey is that in a closed development with a design team this kind of rough work would never be seen by public, as the actual need and possible solutions would be evaluated, discussed, and resolved way earlier in the process. Since Godot is not developed this way, putting stuff out there and then iterating on it is the only viable way to move things forward. So it's not a mistake to author a change and then involve everyone to help iterate on it.

Because I get a feeling that you suggest that it has been a mistake in the first place. So I tried to explain that sensibilities are a bit different when developing open source tools.

> I think you and I are attacking this problem from different angles. I get the feeling you see this from a technical perspective – the area that happened to be available for the logo is so small that the original logo won't look good. So we need to fix it with a logo that can be placed within the pixels we have available. I see this from a designer's perspective – We have an established logo, let's make enough space for it so it can be presented in a way that looks good and is recognizable.

The need to fix the logo is a long-standing one and is not limited to this redesign of the project manager. Many designers mentioned through years problems with the current logo. The communication team of the project also struggled with the practical application of the logo. Again, the thread linked above in this discussion is by an independent designer who happened to illustrate the same problems we've experienced in the project.

The color palette is off, the lines are arbitrary and messy. The lack of the outline makes it hard to use in a lot of mainstream cases and in the same context as other applications in various environments. The monochrome version looks like a skull no matter how we adjust it. The logo doesn't scale despite being used at sizes as small as 8-12 pixels (the app icon).

All these problems have been established before my PR. And while I have no deep feelings towards the roughly made attempt that I've produced, it is IMO a valuable one and is showing how we can evolve the logo to work better at scale without drastic changes to it. I don't want drastic changes to it. But we also cannot NOT change it going forward. It just doesn't work in a lot of places where the project needs it to work.

-----

So all in all, I don't think the existing variation should be treated as a holy cow. I would welcome experienced designers to try and tackle this problem by refining the logo. This has been done before, and we do need it now regardless of the project manager. You can get in touch with @coppolaemilio who tries to build a design team to see which needs the logo has to cover.

And while, yes, I do approach the problem not as a designer, I got positive feedback from designers, such as Kenney, on this attempt. So far I've seen a wide variety of opinions on the proposed design. Some people like everything about it, some think that the logo is excessive and should be gone. Some see problems with how much attention it attracts but are okay otherwise, and some, like you, mention that we shouldn't ever modify existing logos because it's a cardinal sin in graphics design (and that includes my frivolous removal of the "game engine" subtitle).

My main goal with my comments here is to explain the rationale behind the work done on the logo and to highlight some problems with the suggestion. I didn't feel like that point of view was represented in this thread before me. But ultimately, everyone is welcome to make a PR that changes the current state.

**Comment by peterhoglund** [comment]
>like you, mention that we shouldn't ever modify existing logos because it's a cardinal sin in graphics design

I think you are being hyperbole or intentionally misunderstanding me. I don't say we can never make changes to the logo. The logo is flawed and I'd love if we adjust it to be more professionally constructed. What I am saying is, you can't just go and do random changes to a logo that distorts its appearance.

But I give up. I have provided you with tons of arguments to why this is a bad idea, I've provided suggestions on how to go around the problem and I am obviously talking to the right person. You clearly do not want to consider this so there is nothing more I can do.

**Comment by YuriSizov** [comment]
> I think you are being hyperbole or intentionally misunderstanding me.

Well, listen, I kind of feel the same about your messages now. I tried my best to explain how this change was not done on a whim and what discussions and context preceded it. It was also approved by multiple people representing the project, even if ever so reluctantly, and I tried to explain why it was okay to go forward with it despite consideration that you voiced. I explained what can be done next. 

You are of course entitled to a different opinion,  and I want to encourage you to try and improve the design if you feel strongly about it.

PS. I specifically avoided deconstructing your mockups because I don't think this is important to the discussion. I can explain what issues I see with them, if you want, but I would rather not.

**Comment by peterhoglund** [comment]
> Well, listen, I kind of feel the same about your messages now.

I didn't mean I think you intentionally misunderstand me in general, I only meant your comment about me never wanting to change the logo, because that is not at all what I am saying. 

Anyway, if the community has accepted this I will not bang my head bloody. I do think the changes to the Project Manager are great, just not the logo change.

**Comment by hakro** [comment]
I must agree with OP here. The new Project Manager layout looks great. But the first time I opened it, the logo immediately jumped as "out of place", and actually looked quite creepy in this particular situation.

That's even more accentuated by the fact that KDE (at least in its stock form), includes the logo in the window title bar.

And finally, clicking that logo shows the `Thanks from the Godot Community!` screen, which also shows the usual Godot blue logo, twice. So there the lack of consistency is pretty obvious.

![Screenshot_20240213_201242](https://github.com/godotengine/godot-proposals/assets/3624853/fdcb25a4-72e1-4c8b-b541-d607eab31a2a)


Cheers !

**Comment by Deozaan** [comment]
I would agree that the monochrome logo looks out of place and inconsistent in this scenario, surrounded by all the regularly colored logos. While reading this thread I kept wondering why any logo needs to be there at all when there's one in the title bar just above it, but then I remembered that this logo is actually a button that is used to access what is usually found in `Help -> About` menus in other applications, as shown in the previous comment.

With the understanding that we need to allow users some way to access the "about" window, I like the alternatives proposed with the logo/button on the right. Especially the bigger one in the bottom-right corner. There would even be enough space there to put "game engine" back under the logo is wanted, though I don't feel too strongly about that particular detail.

Another alternative is to just move past the idea of using the logo for this button and instead place a "question/help" or "information" icon in the upper-right corner (to the right of the Settings button) that will pop up the "about" window when clicked.
